internal/kafka/internal/services/quota/quota_management_list_service_test.go

```diff
@@ -462,7 +461,45 @@ func Test_QuotaManagementListReserveQuota(t *testing.T) {
 				InstanceType:   tt.args.instanceType.String(),
 			}
 			_, err := quotaService.ReserveQuota(kafka, tt.args.instanceType)
-			gomega.Expect(tt.wantErr).To(gomega.Equal(err))
+			Expect(tt.wantErr).To(Equal(err))
+		})
+	}
+}
+func Test_DefaultQuotaServiceFactory_GetQuotaService(t *testing.T) {
+	type fields struct {
+		QuoataServiceContainer map[api.QuotaType]services.QuotaService
+	}
+	type args struct {
+		quoataType api.QuotaType
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		args   args
+		want   services.QuotaService
```

Comment:
Variable names: want, want1 are not self explanatory, can we make them more context specific. Always varibale names like x,x1 should be avoided.